contentcuration/contentcuration/templates/base.html

```diff
@@ -94,6 +92,8 @@
         </script>
         {% endif %}
         {% endcache %}
```

Comment:
I moved css bundle out of cache block but I am actually not sure if this change is required. I can just remember that I experienced random issues with assets not being loaded at all and this seemed to help.